elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```diff
@@ -85,6 +112,21 @@ protected void configure() {
             protected void configure() {
                 Map<String, Swagger> swaggerDocs = settings.enableSwagger();
                 if (!swaggerDocs.isEmpty()) {
+                    // Include the async models in swagger docs
+                    if(settings.enableAsync()) {
+                        EntityDictionary dictionary = new EntityDictionary(new HashMap());
+                        dictionary.bindEntity(AsyncQuery.class);
+                        dictionary.bindEntity(AsyncQueryResult.class);
+                         
+                        Info info = new Info().title("Async Service").version("1.0");
+
+                        SwaggerBuilder builder = new SwaggerBuilder(dictionary, info);
+
+                        Swagger swagger = builder.build().basePath("/api/v1");
```

Comment:
Most of the other APIs have a config setting for the base path.

Reply:
As discussed no config setting exists will add a new one.